specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json

```diff
@@ -1189,11 +1215,12 @@
       ]
     },
     "create_session_settings": {
+      "description": "Settings of the session to be created.",
       "properties": {
         "maxLeaseTimeMinutes": {
           "example": 60,
           "type": "integer",
-          "description": "The time in minutes the session will run after reaching the 'Ready' state."
+          "description": "The time in minutes the session will run after reaching the 'Ready' state. It has to be between 0 and 1440."
```

Comment:
"It has to be between 0 and 1440."
For integers you can add min/max attributes in addition, not?

Reply:
I'd rather not make any API changes in this PR (adding min & max would constitute as changing the API spec AFAIK), and I'm not sure whether this wasn't limited intentionally or not - e.g. if we want to be flexible in reducing/expanding this number. I decided to write the current limits in the docs as it would be useful for customers to know, but would also allow us to change easily if needed.